### ISSUE TITLE
Linux OAuth2 response cannot be used.

### ISSUE THREAD

**Issue by pdf** [microsoft/vscode-pull-request-github#622]
- Extension version: v0.2.1
- VSCode Version: v1.28.2
- OS: Linux (Arch)

Steps to Reproduce:

1. Add URL handler for `vscode://` scheme (this should be done in vscode by default, but that's another discussion)
2. Attempt to authenticate via the OAuth2 flow
3. When vscode receives the callback URL, it displays an error dialog the width of the URL (typically this overflows the screen) stating:

```
Extension 'Github Pull Requests' is not loaded.
Would you like to reload the window to load
the extension and open the URL?
```

Clicking `Cancel` does as expected and cancels the OAuth2 flow, clicking `Reload Window and Open` just repeats this message forever.

![screenshot_20181026_133742](https://user-images.githubusercontent.com/146192/47541606-91ec4d00-d925-11e8-990d-c2ebfc6851b6.png)

Related to #573

**Comment by mcamou** [comment]
The extension was working for me until the latest update (extension v0.3.1). I'd made the extension an essential part of my workflow, and now it's not working. I tried to add the `githubPullRequests.hosts` setting manually, but I don't see all PRs. For example, in one our the private repos under "All pull requests" we use I see one single PR which has been closed for a long time, while there are 7 open PRs.
